src/GPU/fix_gpu.cpp

```diff
@@ -53,51 +53,57 @@ extern double lmp_gpu_update_bin_size(const double subx, const double suby, cons
                                       const int nlocal, const double cut);
 
 static const char cite_gpu_package[] =
-  "GPU package (short-range, long-range and three-body potentials):\n\n"
+  "GPU package (short-range, long-range and three-body potentials): doi:10.1016/j.cpc.2010.12.021, doi:10.1016/j.cpc.2011.10.012, doi:10.1016/j.cpc.2013.08.002, doi:10.1016/j.commatsci.2014.10.068, doi:10.1016/j.cpc.2016.10.020, doi:10.3233/APC200086\n\n"
   "@Article{Brown11,\n"
-  " author = {W. M. Brown, P. Wang, S. J. Plimpton, A. N. Tharrington},\n"
-  " title = {Implementing Molecular Dynamics on Hybrid High Performance Computers - Short Range Forces},\n"
-  " journal = {Comp.~Phys.~Comm.},\n"
+  " author = {W. M. Brown and P. Wang and S. J. Plimpton and A. N. Tharrington},\n"
+  " title = {Implementing Molecular Dynamics on Hybrid High Performance Computers---Short Range Forces},\n"
+  " journal = {Comput.\\ Phys.\\ Commun.},\n"
   " year =    2011,\n"
   " volume =  182,\n"
-  " pages =   {898--911}\n"
+  " pages =   {898--911},\n"
+  " doi =     {10.1016/j.cpc.2010.12.021}\n"
   "}\n\n"
   "@Article{Brown12,\n"
   " author = {W. M. Brown, A. Kohlmeyer, S. J. Plimpton, A. N. Tharrington},\n"
   " title = {Implementing Molecular Dynamics on Hybrid High Performance Computers - Particle-Particle Particle-Mesh},\n"
-  " journal = {Comp.~Phys.~Comm.},\n"
+  " journal = {Comput.\\ Phys.\\ Commun.},\n"
   " year =    2012,\n"
   " volume =  183,\n"
+  " doi =     {10.1016/j.cpc.2011.10.012},\n"
   " pages =   {449--459}\n"
   "}\n\n"
   "@Article{Brown13,\n"
   " author = {W. M. Brown, Y. Masako},\n"
-  " title = {Implementing Molecular Dynamics on Hybrid High Performance Computers – Three-Body Potentials},\n"
-  " journal = {Comp.~Phys.~Comm.},\n"
+  " title = {Implementing Molecular Dynamics on Hybrid High Performance Computers---Three-Body Potentials},\n"
+  " journal = {Comput.\\ Phys.\\ Commun.},\n"
   " year =    2013,\n"
   " volume =  184,\n"
-  " pages =   {2785--2793}\n"
+  " pages =   {2785--2793},\n"
+  " doi =     {10.1016/j.cpc.2013.08.002},\n"
   "}\n\n"
   "@Article{Trung15,\n"
   " author = {T. D. Nguyen, S. J. Plimpton},\n"
-  " title = {Accelerating dissipative particle dynamics simulations for soft matter systems},\n"
-  " journal = {Comput.~Mater.~Sci.},\n"
+  " title = {Accelerating Dissipative Particle Dynamics Simulations for Soft Matter Systems},\n"
+  " journal = {Comput.\\ Mater.\\ Sci.},\n"
   " year =    2015,\n"
+  " doi =     {10.1016/j.commatsci.2014.10.068},\n"
   " volume =  100,\n"
   " pages =   {173--180}\n"
   "}\n\n"
   "@Article{Trung17,\n"
   " author = {T. D. Nguyen},\n"
-  " title = {GPU-accelerated Tersoff potentials for massively parallel Molecular Dynamics simulations},\n"
-  " journal = {Comp.~Phys.~Comm.},\n"
+  " title = {{GPU}-Accelerated {T}ersoff Potentials for Massively Parallel Molecular Dynamics Simulations},\n"
+  " journal = {Comput.\ Phys.\ Commun.},\n"
```

Comment:
```suggestion
  " journal = {Comput.\\ Phys.\\ Commun.},\n"
```

Reply:
Fixed this and some other BibTeX issues in the same file